### ISSUE TITLE
ms team app tab auth doesn't work on mobile client 

### ISSUE THREAD

**Issue by MicroAsif** [OfficeDev/microsoft-teams-library-js#216]
Hi there, 

we have made a team tab application which has tab authentication, works on web and desktop but when we're on the phone/tab the popup never shows. 

can you please let us know, if we're missing something  ? 

thanks

**Comment by Gousia-MSFT** [comment]
@MicroAsif Which SDK version are you using? Could you please try using version 1.4 and check if that helps

**Comment by MicroAsif** [comment]
we have updated with the version 1.4 still the popup never shows in team mobile apps. do we need to configure anything specifically for mobile device  in azure active directory ?

**Comment by MicroAsif** [comment]
after updating 1.4 mobile client goes the MSFT login page for the first time but never fired the notify success method, redirects to the application login page again and again.

**Comment by billbliss** [comment]
What is the specific version you are using? It's actually 1.4.1 or above.

**Comment by billbliss** [comment]
In issue #173 it appears you are referring to the same question, and there you are using 1.4.2. Is that true? Let's just use one issue to track this - this one's open and the other isn't, so let's use this one.

**Comment by MicroAsif** [comment]
we're using 1.4.2, the latest one. it doesn't return the notifySuccess result on android.

**Comment by Gousia-MSFT** [comment]
@MicroAsif Could you please share a screenshot of the version of Teams Mobile client you are using? You could get that from Settings->About.

**Comment by MicroAsif** [comment]
hers it is - https://imgur.com/a/ZLNmHDR

**Comment by Gousia-MSFT** [comment]
@MicroAsif Fix for this issue has been rolled out. Could you please check if the issue is resolved?

**Comment by JanOnGit** [comment]
Hi there,
i know this issue is already closed. But I still have this problem. I just created an Azure Web-App application (.NETCore 2.2.0, MVC)  with tabs for teams with authentication. I implemented a silent authentication and SSO like in this example (https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/authentication/auth-silent-aad). On teams desktop the authentication works as expected. On teams mobile the authentication fails (same application). The mobile client goes to login page (adsf) but never closes and it doesn't return the notifySuccess result. I created the application exactly for the silent authentication and SSO on mobile, to test how it works. I use MicrosoftTeams SDK v1.5.2 with ADAL v1.0.17. Desktop client Microsoft Teams Version 1.2.00.26774 (64-bit). It was last updated on 10/8/19. Mobile client Version 1416/1.0.0.2019092603 / Calling Version 2019.32.01.7.

thanks
